application/templates/pvc.yaml

```diff
@@ -1,24 +1,22 @@
-{{- if .Values.persistence }}
-{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
+{{- if and (.Values.persistence).enabled (not .Values.persistence.existingClaim) }}
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
+{{- if .Values.persistence.name }}
+  name: "{{ .Values.persistence.name }}"
+{{- else }}
+  name: {{ template "application.name" . }}-data
```

Comment:
we need to update claim name in deployment.yaml https://github.com/stakater/application/blob/5bc7a996b42206e58c6cd248f916282fa79fa609/application/templates/deployment.yaml#L276

Reply:
great catch @mustafaStakater plz fix it

Reply:
fixed